kerberos.json

```diff
@@ -0,0 +1,126 @@
+{
+  "services": [
+    {
+      "name": "CDAP",
+      "identities": [
+        {
+          "name": "/spnego"
+        },
+        {
+          "name": "/hdfs"
+        },
+        {
+          "name": "cdap",
+          "principal": {
+            "value": "cdap@${realm}",
+            "type" : "user",
+            "configuration": "cdap-env/cdap_principal_name",
+            "local_username": "cdap"
+          },
+          "keytab": {
+            "file": "${keytab_dir}/cdap.headless.keytab",
```

Comment:
I meant to say, the others use cdap.service.keytab (also file name).  but i see now that this one is the user keytab, not the service keytab